core/data/core.joke

```diff
@@ -3202,6 +3202,11 @@
     :doc "True while a verbose load is pending"}
   *loading-verbosely* false)
 
+(defonce ^:dynamic
+  ^{:private true
+    :doc "A map from namespace reg-exps to the source to load the external dependency from"}
+  *ns-sources* {})
```

Comment:
`*ns-sources*` and `ns-sources` need to be un-mapped [here](https://github.com/candid82/joker/blob/master/core/data/linter_cljx.joke#L382) for linting purposes.

Reply:
When I unmapped `*ns-sources*` at that location I ended up getting the following in other tests:
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
```

I can provide full backtrace if that helps, but it wasn't obvious what the error was here. I didn't see any other ear-muffed variables being unmapped at that location.

Reply:
I'll look into it later, no big deal.